### DISCUSSION TITLE
Flex Users

### DISCUSSION THREAD

**Discussion by sharfatehi** [twilio-labs/paste#1110] [Office hours] [answered]
**The specific questions I have are...**

- Are my patterns consistent in Flex Users
- Are there any existing patterns for adding/removing items from a table, if not does my add/remove pattern fit with Flex/Paste patterns 

**Background**: I am preparing for a UX review and I want to make sure my patterns are consistent. I would also like to discuss a new pattern I am introducing around adding/removing items to a table.

**Stage of design**: Hi-Fidelity
**Research**: 
**Links to mockups or prototype**: https://twilio.invisionapp.com/share/HGZRATM3TKR#/441714656_Dashboard

**Comment by corinne-nagel** [comment, marked as answer]
Hi @sharfatehi - 

• Could you give more specifics on which patterns you're asking about for consistency? You can see [our current and planned patterns ](https://paste.twilio.design/patterns)here.

• The [Create](https://paste.twilio.design/patterns/create) and [Delete patterns](https://paste.twilio.design/patterns/delete/) exist currently, but I would encourage you to see how this is being handled in other parts of the product; this would be good opportunity to take it to your crit pod to help you iterate on and gather feedback from others in Flex. We can give feedback on this exploration/research and see what components might be needed/if any are missing or any specific accessibility requirements that will need to be considered. 

If you'd like to contribute a pattern 🙏,  or have more questions, please [reference this guide](https://paste.twilio.design/patterns/contribute) and attend [DSYS office hours](https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit#gid=800758837) so we can help and get you started on the contribution process.

**Reply by serifluous** [reply]
+1 to @corinne-nagel . We also know there's already a lot of great Patterns work going on in the Flex team. Would love to chat more about how we can get that documentation on the Paste site, too!

**Reply by sharfatehi** [reply]
Hi @corinne-nagel,

Thanks for the links, I might have missed it but I am not seeing what the flow looks like when you click on the "add" buttons. I am also curious about what you do when you have added something and want to delete it. I have been working on a pattern and it has been reviewed extensively in Crits with the Flex team. I have signed up for DSYS office hours, which I'll be attending later today. We haven't added this to our patterns document that our team is working on - I wanted to review with the paste team first. 

@serifluous - I'd be happy to discuss and collaborate on how to get our documentation on the Paste site. Feel free to set something up with me any time :)

Sharmeen
